test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -744,10 +749,221 @@ def __are_requirements_matching(param_dic=None, test=None):
+                _msg = "hostname- " + hostname + ": used disk is "
+                self.logger.warning(_msg + str(used_disk_percent) + "%")
+            elif used_disk_percent > 95:
+                _msg = "hostname- " + hostname + ": used disk is "
+                self.logger.error(_msg + str(used_disk_percent) + "%")
+                self.hardware_report_timer.cancel()
+                raise PbsSystemError(msg=_msg+str(used_disk_percent)+"%")
+            # checks core files in mom_priv
+            pbs_conf = du.parse_pbs_config(hostname)
+            mom_priv_path = os.path.join(pbs_conf["PBS_HOME"],
+                                         "mom_priv")
+            mom_priv_files = du.listdir(hostname,
+                                        mom_priv_path, sudo=True,
+                                        fullpath=False)
+            for filename in mom_priv_files:
+                if filename.startswith("core"):
+                    _msg = "hostname- " + hostname + ": core file exist at "
+                    self.logger.warning(_msg + mom_priv_path)
+                    self.hardware_report_timer.cancel()
+                    raise PbsSystemError(msg=_msg + mom_priv_path)
+            # checks core files in server_priv
+            server_priv_path = os.path.join(pbs_conf["PBS_HOME"],
+                                            "server_priv")
+            server_priv_files = du.listdir(hostname,
+                                           server_priv_path, sudo=True,
+                                           fullpath=False)
+            for filename in server_priv_files:
+                if filename.startswith("core"):
+                    _msg = "hostname- " + hostname + ": core file exist at "
+                    self.logger.warning(_msg + server_priv_path)
+                    self.hardware_report_timer.cancel()
+                    raise PbsSystemError(msg=_msg + server_priv_path)
+            # checks core files in sched_priv
+            sched_priv_path = os.path.join(pbs_conf["PBS_HOME"],
+                                           "sched_priv")
+            sched_priv_files = du.listdir(hostname,
+                                          sched_priv_path, sudo=True,
+                                          fullpath=False)
+            for filename in sched_priv_files:
+                if filename.startswith("core"):
+                    _msg = "hostname- " + hostname + ": core file exist at "
+                    self.logger.warning(_msg + sched_priv_path)
+                    self.hardware_report_timer.cancel()
+                    raise PbsSystemError(msg=_msg + sched_priv_path)
+            # checks core files in user home directory
+            ptl_users = ['pbsuser1', 'pbsuser2', 'pbsuser3', 'pbsuser4',
+                         'pbsuser5', 'pbsuser6', 'pbsuser7', 'pbstest',
+                         'pbsroot', 'pbsadmin']
+            for user in ptl_users:
+                cmd = "getent passwd " + user
```

Comment:
once you use PBS_ALL_USERS you don't need to do this as PbsUser() has `.home` an attribute which will give you home dir path for that user